### ISSUE TITLE
Unable to reply email with attachments

### ISSUE THREAD

**Issue by imleaver** [RainLoop/rainloop-webmail#1350]
**RainLoop version, browser, OS:**
Version: 1.10.5.192
Browser: Chrome Version 56.0.2924.87 (64-bit)
OS: Ubuntu 16.04 LTS 64bits

**Expected behavior and actual behavior:**
When I was trying to reply or reply-all an email, there are no attachments in the attachment area. But it works with forward function.

This is same goes to https://github.com/RainLoop/rainloop-webmail/issues/1091 , but this time is for reply function.

**Steps to reproduce the problem:**
Just ask someone email you with attachment and when you reply the email, there will be no attachments

**Logs or screenshots:**
This is the email
![image](https://cloud.githubusercontent.com/assets/1909268/23402446/90f35ba8-fde6-11e6-9366-87881246e65e.png)

When I click reply or reply all, there will be no attachments in the section.
![image](https://cloud.githubusercontent.com/assets/1909268/23402474/b34516e2-fde6-11e6-9731-ae7ffefe4a50.png)

**Comment by megadr01d** [comment]
Yeah, why would you want to send the attachments to the guy who sent you the attachments? I believe some clients may have the option to include attachments when replying, but the default is off and I can't see a user-case scenario where this would be useful, not to mention it's a waste of bandwidth to keep sending attachments back and forth.
